src/module/interact.py

```
@@ -0,0 +1,170 @@
+                              priority=50, block=True, rule=to_me())
+today_contests_2 = on_command("今日比赛", aliases={"今日的比赛", "今天比赛", "今天的比赛"}, priority=50, block=True,
+                              rule=to_me())
+recent_contests_1 = on_command(("contest", "recent"),
+                               aliases={("contests", "recent"), ("比赛", "近期"), ("比赛", "最近")}, priority=50,
+                               block=True, rule=to_me())
+recent_contests_2 = on_command("近期比赛", aliases={"最近比赛", "最近的比赛"}, priority=50, block=True, rule=to_me())
+
+
+@today_contests_1.handle()
+@today_contests_2.handle()
+async def reply_today_contests(event: Event, message: Message = CommandArg()):
```

Ruff (0.12.2)

Do not perform function call `CommandArg` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)